docs/guides/quickstart-dfp.md

```diff
@@ -0,0 +1,258 @@
+---
+page_title: "Managing Policy Based DFP service using the BloxOne Terraform Provider"
+subcategory: "Guides"
+description: |-
+  This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage the DFP Service , Security Policies and various resources associated to it.
+---
+
+# Managing Policy Based DFP service using the BloxOne Terraform Provider
+
+This guide provides step-by-step instructions for using the BloxOne Terraform Provider to manage the DFP Service , Security Policies and various resources associated to it.
+
+## Configuring the Provider
+
+The provider needs to be configured with an API key and the URL of the Infoblox Cloud Services Portal (CSP). You can get the API Key from the Infoblox Cloud Services Portal (CSP) by following the steps outlined in this guide - [Configuring User API Keys](https://docs.infoblox.com/space/BloxOneCloud/35430405/Configuring+User+API+Keys).
+
+Create a directory for the Terraform configuration and create a file named `main.tf` with the following content:
+
+````terraform
+terraform {
+  required_providers {
+    bloxone = {
+      source  = "infobloxopen/bloxone"
+      version = ">= 1.0.0"
+    }
+  }
+  required_version = ">= 1.5.0"
+}
+
+provider "bloxone" {
+  csp_url = "https://csp.infoblox.com"
+  api_key = "<BloxOne API Key>"
+}
+````
+
+!> Warning: Hard-coded credentials are not recommended in any configuration file. It is recommended to use environment variables.
+
+You can also use the following environment variables to configure the provider:
+`BLOXONE_CSP_URL` and `BLOXONE_API_KEY`.
+
+Initialize the provider by running the following command. This will download the provider and initialize the working directory.
+
+```shell
+terraform init
+```
+
+## Configuring Resources
+
+### BloxOne Host on AWS with DFP service
+
+As the first step, you will configure a BloxOne Host on AWS with DFP service.
+
+You will use the following module to create it:
+- [bloxone_infra_host_aws](https://github.com/infobloxopen/terraform-provider-bloxone/tree/master/modules/bloxone_infra_host_aws)
+
+The module requires the [AWS terraform provider](https://registry.terraform.io/providers/hashicorp/aws/latest) to be configured.
+To configure the AWS provider, add the following code to your main.tf:
+
+````terraform
+provider "aws" {
+  region     = "us-west-2"
+  access_key = "my-access-key"
+  secret_key = "my-secret-key"
+}
+```` 
+
+!> Warning: Hard-coded credentials are not recommended in any configuration file. It is recommended to use the AWS credentials file or environment variables.
+
+You can also use the following environment variables to configure the provider:
+`AWS_REGION`, `AWS_ACCESS_KEY_ID`, and `AWS_SECRET_ACCESS_KEY`.
+
+To create an EC2 instance with DHCP service, you will need to have the following information:
```

Comment:
Why do we mention about DHCP service here ?

Reply:
Also, why are we creating the host with DHCP service in DFP example ?

Reply:
This is a typo . Has been modified.